gradoop-flink/src/test/java/org/gradoop/flink/algorithms/gelly/clusteringcoefficient/GellyClusteringCoefficientTestBase.java

```diff
@@ -42,11 +35,13 @@
    */
   public GellyClusteringCoefficientTestBase() {
```

Comment:
Don't use a constructor here. Use the `@Before` annotation to define a setUp method where the loader is filled.  Remove also each constructor of it's sub classes. Each test of the sub class will call this setUp method automatically.

Reply:
Changed it